quantum/keymap_extras/keymap_slovak.h

```diff
@@ -141,9 +141,10 @@
  */
 // Row 1
 #define SK_TILD ALGR(SK_PLUS) // ~
-#define SK_CIRC ALGR(SK_LCAR) // ^ (dead)
-#define SK_BREV ALGR(SK_SCAR) // ˘ (dead)
-#define SK_RNGA ALGR(SK_CCAR) // ° (dead)
+#define SK_CNAL ALGR(SK_LCAR) //ˇ (dead) as in caron alternative
+#define SK_CIRC ALGR(SK_SCAR) // ^ (dead)
+#define SK_BREV ALGR(SK_CCAR) // ˘ (dead)
+#define SK_DGAL ALGR(SK_TCAR) // ° (dead) as in degree alternative
```

Comment:
```suggestion
#define SK_CIRC ALGR(SK_SCAR) // ^ (dead)
#define SK_BREV ALGR(SK_CCAR) // ˘ (dead)
```
These are not needed as they're already defined above.

Reply:
yes, but for example, if i set up a key override for the normal caron key to something else, and i still want to be able to type the caron, i can do it this way. since this is already in the os layout, i don't see a point in not telling the user you can also get the caron like this. that's why i added the names ending as AL, as in ALTGR or ALTERNATIVE. would this be a problem? if yes, why?

Reply:
This is the convention I've gone with. If a character is available in a "less-modified" way, we favour that one and ignore the other(s) for simplicity. It is not that common anyway. The exception is when the deadness of the key differs.

Reply:
well, okay.